src/pages/EventPage.jsx

```diff
@@ -0,0 +1,381 @@
+    // getEventId(eventId);
+  }, []);
+
+  // const handleSelectButton = () => {
+  //   setShowSelect(true);
+  //   setIsSelectButton(false);
+  //   setIsCreateButton(false);
+  // };
+
+  // const handleGoBackButton = () => {
+  //   setShowSelect(false);
+  //   // setIsCreateButton(true);
+  //   // setIsSelectButton(true);
+  //   setSelectedEvents([]);
+  // };
+
+  // const SelectButton = () => {
+  //   return (
+  //     <>
+  //       <Button
+  //         style={{ backgroundColor: 'white' }}
+  //         onClick={() => handleSelectButton()}
+  //         fontSize="20px"
+  //         height={'50px'}
+  //       >
+  //         Select
+  //       </Button>
+  //     </>
+  //   );
+  // };
+
+  const handleLocationChange = event => {
+    const selectedLocation = event.target.value;
+    console.log(selectedLocation);
+    setLocation(selectedLocation);
+  };
+
+  const handleDateChange = event => {
+    const selectedDate = event.target.value;
+    console.log(selectedDate);
+    setDate(selectedDate);
+  };
+
+  const getLocationOptions = () => {
+    return locations.map((location, index) => (
+      <option key={location + index.toString()} value={location}>
+        {location}
+      </option>
+    ));
+  };
+
+  const getDateOptions = () => {
+    return dates.map((date, index) => (
+      <option key={date + index.toString()} value={date}>
+        {date}
+      </option>
+    ));
+  };
+
+  // const ArchiveButton = ({ id }) => {
+  //   return (
+  //     <>
+  //       <Button
+  //         style={{ backgroundColor: '#FFABAB', borderRadius: '30px' }}
+  //         onClick={() => archiveEvents(id)}
+  //       >
+  //         <Box padding={3} fontSize={'lg'} display="inline-flex" gap={10}>
+  //           Archive Event(s)
+  //         </Box>
+  //       </Button>
+  //     </>
+  //   );
+  // };
+
+  // ArchiveButton.propTypes = {
+  //   id: PropTypes.number,
+  // };
+
+  // const DeselectButton = () => {
+  //   return (
+  //     <>
+  //       <Button
+  //         style={{ backgroundColor: 'white', borderRadius: '0px' }}
+  //         fontSize="20px"
+  //         height={'50px'}
+  //         onClick={() => handleGoBackButton()}
+  //       >
+  //         Deselect All
+  //       </Button>
+  //     </>
+  //   );
+  // };
+
+  useEffect(() => {
+    if (!fuse) {
+      return;
+    }
+    console.log(name);
+    let ands = [];
+    if (name) ands.push({ name: name });
+    if (location) ands.push({ location: location });
+    if (date) ands.push({ date: date });
+
+    let result;
+    if (ands.length > 0) {
+      const fuseResult = fuse.search({ $and: ands });
+      console.log(fuseResult);
+      // If we want to filter by score:
+      // result = fuseResult.filter(item => item.score <= 0.5).map(item => item.item);
+      result = fuseResult.map(item => item.item);
+    } else result = events;
+    console.log(result);
+    setDisplayEvents(result);
+  }, [name, location, date, fuse]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'events'. Either include it or remove the dependency array.